### ISSUE TITLE
JIP is currently broken with Eden update.

### ISSUE THREAD

**Issue by perfk** [dklollol/Olsen-Framework-Arma-3#52]
If a mission is created with the Eden editor (3d Editor) will break JIP.
Tested with a fresh version downloaded from github.

Jips does not get their gear when spawning in.

I also tried with the respawning function. If a player was not a jip, would gear work fine. But JIPS would not get new gear on respawning. I'm guessing jips somehow is not getting the init.sqf? and therfore not getting the FW_RespawnEh = player addEventHandler ["Respawn", {"" call FNC_SpectatePrep;}];

**Comment by Reyfin** [comment]
Does this happen when  using the framework gear stuff or in general?

**Comment by perfk** [comment]
It seems to be an issue with the Units init field.
 
From my post on UO
Hi all,
 
So it started with Gearscripts not working when players jiped in. After extensive testing have Blackhawk and I concluded that the Eden editor malfunctioning with jip units init field.
 
In short: If you have any commands in the Units Init field. Will this not be used when someone jips into this unit.
 
Simple test to prove this:
Make mission.
Add 2 slots
Add hint "Test 1" to unit 1.
Add hint "Test 2" to unit 2.
Now start a server up and join unit 1. It will say "Test 1" as a hint.
Now get someone to jip into unit 2. It will say "Test 1" as a hint for that player as well.
 
Now do the same but in the 2D editor, and you will see that here is unit 1 being show Test 1, and unit 2 is being shown Test 2.
 
The only thing that is being used from the Jipped unit is the units NAME.
 
In short. Olsens Framework (and most likely any other that uses the units init field) will be broken if you make a mission using the Eden editor.
 
I can't report this, as the bugtracker is currently down.

**Comment by Starfox64** [comment]
Fixed as per the latest changelog https://dev.arma3.com/post/spotrep-00053
